app/scripts/utils/localStorage.js

```diff
@@ -31,11 +30,11 @@ function doOperation (op) {
     if (op === 'getItem') {
       try {
         result = JSON.parse(result)
-      } catch(e) {}
+      } catch (e) {}
```

Comment:
in another pr, trying not to change the components unless I have to this round (just linting here)